docs/docs/requirements.md

```diff
@@ -0,0 +1,6 @@
+# requires
+
+- Python 3.6+
+- Jinja
```

Comment:
Would good you set the package version, Example:

Flask>1.0.0

Reply:
Done!